src/framebuffer.ml

```diff
@@ -50,15 +52,16 @@ let draw_circle (x : int) (y : int) (r : float) (col : int) (buffer : t) =
   done
 
 let filled_circle (x : int) (y : int) (r : float) (col : int) (buffer : t) =
+  buffer.dirty <- true;
```

Comment:
Are there any scenarios where I can call filled_circle and not change the buffer? What if I draw a circle that's off the screen? A circle that is 0 size? In those cases I'd not expect the dirty bit to be set, but this means it always will be.